Riot/Modules/LocationSharing/UserLocationService.swift

```diff
@@ -111,34 +115,33 @@ class UserLocationService: UserLocationServiceProtocol {
         })
     }
     
+    private func getExistingDeviceBeaconSummaries() -> [MXBeaconInfoSummaryProtocol] {
+        guard let userId = self.session.myUserId else {
+            return []
+        }
+        
+        return self.session.locationService.getBeaconInfoSummaries(for: userId).filter { summary in
+            return self.isDeviceBeaconInfoSummary(summary) && summary.isActive
+        }
+    }
+    
+    private func setupDeviceBeaconSummaries() {
+        let existingDeviceBeaconInfoSummaries = self.getExistingDeviceBeaconSummaries()
+        
+        self.deviceBeaconInfoSummaries = existingDeviceBeaconInfoSummaries
```

Comment:
I think you can get rid of the local `existingDeviceBeaconInfoSummaries`. The `self.`s too, here and in a lot of other places.